crates/ironrdp-graphics/src/pointer.rs

```diff
@@ -0,0 +1,379 @@
+use ironrdp_pdu::{
+    cursor::ReadCursor,
+    pointer::{ColorPointerAttribute, LargePointerAttribute, PointerAttribute},
+};
+use thiserror::Error;
+
+/// References:
+/// Drawing pointers: https://learn.microsoft.com/en-us/windows-hardware/drivers/display/pointer-drawing
+/// Drawing color pointers: https://learn.microsoft.com/en-us/windows-hardware/drivers/display/drawing-color-pointers
+/// Drawing monochrome pointers https://learn.microsoft.com/en-us/windows-hardware/drivers/display/drawing-monochrome-pointers
+///
+///
+/// andMask is always 1bpp.
+/// andMask == 0 -> Copy pixel from xorMask
+/// andMask == 1, xorMask == 0 -> Transparent pixel
+/// andMask == 1, xorMask == 1 -> Pixel is inverted
+///
+/// Input: andMask, xorMask, lastFrame -> Output: pixelColor, invertionMask
```

Comment:
This is considered a docstring for the `PointerError` enum below. You can either promote it to a module docstring by putting it at the top and using the `//!` prefix instead of `///`, or if you don’t want this to be module-level documentation (i.e.: showing in the rustdoc), use the simple comment prefix `//` as done in https://github.com/Devolutions/IronRDP/blob/2cb0b476b5a6c31b19915de9f1167bd5c54f5eb8/crates/ironrdp-pdu/src/mcs.rs#L10

Reply:
Yeah, I added that comment temporarily, I need to change it and make it more informative either way